tests/e2e/testdata/scenarios/powertool_set_bucket_location/script.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: storage.cnrm.cloud.google.com/v1beta1
+kind: StorageBucket
+metadata:
+  labels:
+    label-one: "value-one"
+  name: storagebucket-${uniqueId}
+spec:
+  versioning:
+    enabled: false
+  lifecycleRule:
+    - action:
+        type: Delete
+      condition:
+        age: 7
+
+---
+
+kind: RunCLI
```

Comment:
I suggest we'd document this special usage or make it more generic. WDYT?

Reply:
Done: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1893/commits/976ddf13aff06b5a31d0230119696387ddd7c534